api/v1beta1/grafanafolder_types.go

```diff
@@ -111,3 +116,18 @@ func (in *GrafanaFolder) GetTitle() string {
 
 	return in.Name
 }
+
+func (in *GrafanaFolder) GetResyncPeriod() time.Duration {
+	if in.Spec.ResyncPeriod == "" {
+		in.Spec.ResyncPeriod = DefaultResyncPeriod
```

Comment:
Updated the default value to be `5m`, which is the same with the GrafanaDashboard and GrafanaDatasource. Note that the resync period of GrafanaFolder previously was `10s`. If we want to keep this the same as before, let me know and i'll update it to be `10s`.

Reply:
No, `5m` is ok, it should be the same for all resources. Thanks Jameel!